host/py_module/py_bindings.cpp

```diff
@@ -462,29 +476,39 @@ std::shared_ptr<CNNHostPipeline> create_pipeline(
             {"_streams", json::array()}
         };
 
+        json_config_obj["camera"]["rgb"]["resolution_w"]  = config.rgb_cam_config.resolution_w;
         json_config_obj["camera"]["rgb"]["resolution_h"]  = config.rgb_cam_config.resolution_h;
         json_config_obj["camera"]["rgb"]["fps"]           = config.rgb_cam_config.fps;
+        json_config_obj["camera"]["mono"]["resolution_w"] = config.mono_cam_config.resolution_w;
         json_config_obj["camera"]["mono"]["resolution_h"] = config.mono_cam_config.resolution_h;
         json_config_obj["camera"]["mono"]["fps"]          = config.mono_cam_config.fps;
 
-        HostDataReader _blob_reader;
-        int size_blob = 0;
-        if (!config.ai.blob_file.empty())
-        { 
-            if (!_blob_reader.init(config.ai.blob_file))
+        std::string blob_file[] = {config.ai.blob_file, config.ai.blob_file2};
+
+        HostDataReader _blob_reader[num_stages];
+        int size_blob[num_stages];
+        for (int stage = 0; stage < num_stages; stage++)
+        {
+            if (!blob_file[stage].empty())
             {
-                std::cerr << WARNING "depthai: Error opening blob file: " << config.ai.blob_file << "\n" ENDC;
-                break;
+                if (!_blob_reader[stage].init(blob_file[stage]))
+                {
+                    std::cerr << WARNING "depthai: Error opening blob file: " << blob_file[stage] << "\n" ENDC;
+                    break;
```

Comment:
As there was one more loop introduced, the `break` statement no longer does what was intended. Will fix, here and in the other places.